packages/plugin-astro/test/integration.ts

```diff
@@ -17,7 +17,7 @@ interface Execution {
 }
 
 const sandboxSource = fileURLToPath(new URL('./sandbox', import.meta.url))
-const sandbox = resolve(tmpdir(), `orama-plugin-astro-${Date.now()}`)
+const sandbox = process.env.KEEP_SANDBOX ? '/tmp/orama-astro-sandbox' : resolve(tmpdir(), `orama-plugin-astro-${Date.now()}`)
```

Comment:
Will `'/tmp/orama-astro-sandbox'` work on Windows?

Reply:
Good point. It will not. I'll send a little small PR for this. Anyway it's just a debugging flag mostly for the core team so it's not really important.